tests/zkevm/test_worst_compute.py

```diff
@@ -282,3 +283,135 @@ def code_loop_precompile_call(calldata: Bytecode, attack_block: Bytecode):
         raise ValueError(f"Code size {len(code)} exceeds maximum code size {MAX_CODE_SIZE}")
 
     return code
+
+
+@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "gas_limit",
+    [36_000_000],
+)
+def test_worst_jumps(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_limit: int,
+):
+    """Test running a JUMP-intensive contract."""
+    env = Environment(gas_limit=gas_limit)
+    gsc = fork.gas_costs()
+
+    def jump_seq(i):
+        return Op.PUSH2(4 + 5 * i) + Op.JUMP + Op.JUMPDEST
```

Comment:
We could do also:
```suggestion
        return Op.JUMP(Op.ADD(Op.PC, 1)) + Op.JUMPDEST
```
that sounds more robust, but will eat some extra byte and gas (but not that much I think). 

If the cycle count of that change isn't acceptable, we can simplify a bit too with:
```suggestion
        return Op.JUMP(4 + 5 * i) + Op.JUMPDEST
```
which should be equivalent to your code.